app/controllers/games_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def index
+  end
+
+  def create
+    @game = Game.create(game_params)
+    @game.update_attribute(white_player_id, current_user.id)
+    redirect_to root_path
+  end
+
+  def show 
+    @game = Game.find_by_id(params[:id])
+    if @game.blank?
+      render text: 'Not Found', status: :not_found
+    end
+  end
+
+  def update 
+    @game = Game.find(params[:id])
+    @game.update_attributes(black_player_id: current_user.id)
+    redirect_to game_path(@game.id)
+  end
```

Comment:
Good points. I don't think this has really been decided yet. To keep things simple, we can just assume for now that the first player will always be seated as white and the second player will play as black. I'll add a ticket to the icebox for allowing users to select a color when they create a game, and we can work on it if/when we see fit.